kotsadm/pkg/handlers/logout.go

```diff
@@ -19,14 +19,20 @@ func Logout(w http.ResponseWriter, r *http.Request) {
 
 	sess, err := session.Parse(r.Header.Get("Authorization"))
 	if err != nil {
+		// If there is no session, this is not an error
+		if err == ErrEmptySession {
+			JSON(w, 204, "")
+			return
+		}
+
 		logger.Error(err)
 		w.WriteHeader(401)
 		return
 	}
 
 	// we don't currently have roles, all valid tokens are valid sessions
 	if sess == nil || sess.ID == "" {
-		w.WriteHeader(401)
+		JSON(w, 204, "")
```

Comment:
I would think this would return double quotes as the response body rather than no content